src/plugins/cordova-plugin-device-motion/sim-host.js

```diff
@@ -197,47 +208,58 @@ function createCanvas() {
     ThreeDee.setLight(-300, -300, 800);
 
     node.addEventListener('mousemove', function (e) {
-        if (_mouseDown && !_shiftKeyDown) {
-            _offsets.x = (_offsets.x + _oldX - e.offsetX) % 360;
-            _offsets.y = (_offsets.y + _oldY - e.offsetY) % 360;
-
-            _alpha = _alpha || 0;
-
-            // enforce gamma in [-90,90] as per w3c spec
-            _gamma = -_offsets.x;
-            if (_gamma < -90) {
-                _gamma = -90;
+        if (_mouseDown) {
```

Comment:
We should make sure we are not breaking any functionality with these changes. I wonder if we can run the mobile spec tests before and after the change and compare the results. @albertinad, are the tests available for all the supported plugins so far? Can you point us to some documentation on how to run them?